installing/installing_gcp/manually-creating-iam-gcp.adoc

```diff
@@ -5,10 +5,30 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
 
+In environments where the cloud identity and access management (IAM) APIs are not reachable, or the administrator prefers not to store an administrator-level credential secret in the cluster `kube-system` namespace, you can put the Cloud Credential Operator (CCO) into manual mode before you install the cluster.
+
+include::modules/alternatives-to-storing-admin-secrets-in-kube-system.adoc[leveloffset=+1]
+
+.Additional resources
+
+To learn how to rotate or remove the administrator-level credential secret after installing {product-title}, see xref:../../post_installation_configuration/cluster-tasks.adoc#post-install-rotate-remove-cloud-creds[Rotating or removing cloud provider credentials].
+
+See xref:../../authentication/managing_cloud_provider_credentials/about-cloud-credential-operator.adoc[About the Cloud Credential Operator] for a detailed description of all available CCO credential modes and their supported platforms.
+
 include::modules/manually-create-identity-access-management.adoc[leveloffset=+1]
 
 include::modules/admin-credentials-root-secret-formats.adoc[leveloffset=+1]
 
 include::modules/manually-maintained-credentials-upgrade.adoc[leveloffset=+1]
 
 include::modules/mint-mode.adoc[leveloffset=+1]
+
+include::modules/mint-mode-with-removal-of-admin-credential.adoc[leveloffset=+1]
+
+[id="manually-creating-iam-gcp-next-steps"]
+== Next steps
+
+* Install an {product-title} cluster:
+** xref:../../installing/installing_gcp/installing-gcp-default.adoc#installing-gcp-default[Quickly install a cluster] with default options on installer-provisioned infrastructure
```

Comment:
Has the linked file's name changed to [Installing a cluster quickly on Azure](https://deploy-preview-29545--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_azure/installing-azure-default.html#installing-azure-default)?